src/styles/blocks/card.scss

```
@@ -0,0 +1,118 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  width: 198px;
+  height: 406px;
+
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: var(--white-accent);
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+}
+
+.product-card__top {
+  padding: 32px 19px 40px;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__bottom {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px 16px;
+}
+
+.product-card__description {
+  color: var(--main-color);
+  text-align: left;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  padding-bottom: 4px;
```

1. reset default margin for all `p`, `h2` tags (since you're gonna use these tags for text content)
2. to make outer indent you should use margin (no padding)
```suggestion
  margin-bottom: 4px;
```